packages/ui-editable/src/InPlaceEdit/index.tsx

```diff
@@ -125,13 +126,18 @@ class InPlaceEdit extends Component<InPlaceEditProps> {
   // Render a default edit button, an icon button with the edit icon
   // the margin makes room for the focus ring
   static renderDefaultEditButton = ({
-    // @ts-expect-error ts-migrate(7031) FIXME: Binding element 'isVisible' implicitly has an 'any... Remove this comment to see the full error message
     isVisible,
-    // @ts-expect-error ts-migrate(7031) FIXME: Binding element 'readOnly' implicitly has an 'any'... Remove this comment to see the full error message
     readOnly,
-    // @ts-expect-error ts-migrate(7031) FIXME: Binding element 'label' implicitly has an 'any' ty... Remove this comment to see the full error message
     label,
     ...buttonProps
+  }: {
+    isVisible: boolean
+    readOnly?: boolean
+    label: string
+    elementRef?: (el: Element | null) => void
+    onClick: () => void
+    onFocus: () => void
+    onBlur: () => void
```

Comment:
The props will be passed to an IconButton component, so you can type it in a way that it accepts `Partial<IconButtonProps>`, and those extra 3 props we use in our own method.

```suggestion
    }: Partial<IconButtonProps> & {
    label: string
    isVisible: boolean
    readOnly?: boolean
```